keras_nlp/src/utils/transformers_model_utils/hf_common_port.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import einops
+from safetensors import safe_open
```

Comment:
We probably want an import guard for this. Like

https://github.com/keras-team/keras-nlp/blob/9c5f26728e1e3da2a7c36cf91e72a31ec7229188/keras_nlp/src/utils/preset_utils.py#L35-L40

https://github.com/keras-team/keras-nlp/blob/9c5f26728e1e3da2a7c36cf91e72a31ec7229188/keras_nlp/src/utils/preset_utils.py#L150-L154